navit/maptool/osm.c

```diff
@@ -53,6 +53,11 @@ static struct item_bin item;
 
 
 int maxspeed_attr_value;
+double maxheight_attr_value;
```

Comment:
Why "double"? Since the units are converted to e.g. "cm" anyway, there's no need to have floats here. Later the attrs are added as int to the items anyway.

Reply:
My first guess was to use the osm values directly, but when I saw the unit used for the vehicle attributes are in cm I didn't change the type.